scene/gui/button.cpp

```diff
@@ -60,12 +60,6 @@ Size2 Button::get_minimum_size() const {
 			}
 		}
 	}
-
-	Ref<Font> font = get_theme_font(SNAME("font"));
-	float font_height = font->get_height(get_theme_font_size(SNAME("font_size")));
-
-	minsize.height = MAX(font_height, minsize.height);
```

Comment:
Maybe change it to use font height, only if `!xl_text.is_empty()`, instead of removing. When there is text, restricting min size to the text height seems reasonable.

Reply:
That's what `Size2 minsize = text_buf->get_size();` already does.

Reply:
They give different results, `text_buf->get_size();` returns the size of that specific string (depends on the characters used) and `font->get_height(get_theme_font_size(SNAME("font_size")));` returns the general (maximum) height of the font.

Reply:
It's not the same, text-buffer size is a real text size (depends on context and used fallback fonts), so text will fit, but two adjacent buttons can have a bit different height.